cypress/e2e/tests/rbac/architect.test.ts

```diff
@@ -57,7 +59,8 @@ describe(["@tier2"], "Architect RBAC operations", () => {
     };
 
     before("Creating RBAC users, adding roles for them", () => {
-        login();
+        cy.clearLocalStorage();
+        login(adminUserName, adminUserPassword);
```

Comment:
(Related to previous comment) That is what I was referring to, if we clear the local storage, there should be no need to specify the default user and password

Reply:
Still I preferred to put them explicitly, I had an issue which was resolved by this change, so even if now it is not obligatory - I didn't touch it cause it doesn't interfere